src/seedu/addressbook/AddressBook.java

```diff
@@ -377,6 +387,81 @@ public static String executeCommand(String userInputString) {
     private static String getMessageForInvalidCommandInput(String userCommand, String correctUsageInfo) {
         return String.format(MESSAGE_INVALID_COMMAND_FORMAT, userCommand, correctUsageInfo);
     }
+    
+    /**
+     * Edits a person's info (specified by the command args) in the address book.
+     * The entire command arguments string is treated as a string representation of the person to add.
+     *
+     * @param commandArgs full command args string from the user
+     * @return feedback display message for the operation result
+     */
+    
+    private static String executeEditPerson(String commandArgs) {
+        final Optional<String[]> decodeResult = decodePersonFromString(commandArgs);
+
+        // checks if args are valid (decode result will not be present if the person is invalid)
```

Comment:
may be use a method to check if the person whose details are to be edited is instead of using the below conditional and the comment ?